tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backendutils/APIResources.java

```diff
@@ -3,8 +3,8 @@
 public enum APIResources {
 
     GENERATEURLAPI("/submission/generateUrl"),
-    GETCONFIGURATION("/submission/getConfiguration"),
-    EFILINGURL("https://httpbin.org");
+    USERDETAIL("/submission/"),
+    EFILINGURL("http://localhost:3000/efiling?");
```

Comment:
question: why `?` added to the end here

Reply:
Stripping the query parameter excludes '?'. It was added to assert the expected url. Now it's concatenated within the function that is asserting.